src/utils/commandRunner.ts

```diff
@@ -59,12 +59,23 @@ export class CommandRunner {
         `${executable} ${args}`,
         mountPaths
       );
-      runEnv = undefined;
+      runEnv = {};
```

Comment:
At line 64 `runEnv === undefined` will handle` undefined` scenario so assigning `runEnv = {}` is not required. WDYT?